src/service/url-replacements-impl.js

```diff
@@ -451,7 +451,7 @@ export class UrlReplacements {
       return Promise.resolve();
     } else if (metric < 0) {
       // Metric is not yet available. Retry after a delay.
-      return loadPromise(this.win_).then(() => {
+      return loadPromise(this.win_.document).then(() => {
```

Comment:
I think setting up an `onDocumentReady` to add the `win.complete = true` would be a better solution here. Someone else is bound to get tripped up by the same problem.